lib/elixir/lib/exception.ex

```diff
@@ -653,12 +662,18 @@ defmodule FunctionClauseError do
 end
 
 defmodule Code.LoadError do
-  defexception [:file, :message]
+  defexception [file: nil, message: nil, description: "could not load file"]
```

Comment:
sorry @josevalim, I think I was not clear on what I wanted from my side.. My intention was to call these four exceptions that need a file, and pass a message and still display it.